src/datamodule.py

```diff
@@ -17,10 +19,20 @@ def _array_to_torch(filepath: str) -> torch.Tensor:
     """
     # GeoTIFF - Rasterio
     with rasterio.open(fp=filepath) as dataset:
+        # Get image data
         array: np.ndarray = dataset.read()
         tensor: torch.Tensor = torch.as_tensor(data=array.astype(dtype="float16"))
 
-    return tensor
+        # Get spatial bounding box and coordinate reference system in UTM projection
+        bbox: torch.Tensor = torch.as_tensor(  # xmin, ymin, xmax, ymax
+            data=dataset.bounds, dtype=torch.float64
+        )
+        crs: int = torch.as_tensor(data=dataset.crs.to_epsg(), dtype=torch.int32)
+
+        # Get date
+        date: str = pathlib.Path(filepath).name[15:25]  # YYYY-MM-DD format
```

Comment:
Don't like that we have to parse the date from the filename. @yellowcap, I hinted on this at https://github.com/Clay-foundation/model/pull/54#discussion_r1413281596, but would it be possible to save the datetime information in the GeoTIFF's metadata somehow?

Reply:
After the next pipeline run you'll be able to get the date using
```python
dataset.tags()["date"]
```

Reply:
Awesome, can't wait!